kythe/go/languageserver/bin/main.go

```diff
@@ -37,7 +37,7 @@ import (
 var (
 	pageSize = flag.Int("page_size", 0, "Set the default xrefs page size")
 
-	serverAddr = flag.String("server", "localhost:8080",
+	serverAddr = flag.String("server", "0.0.0.0:8080",
```

Comment:
Default should remain `localhost`.  Security by default.  If anything, add documentation to use `:8080` if they want to expose the server to the wider network.

Reply:
Done

Reply:
"localhost:8080" blocks access using `telnet ::1 8080` but allows access `telnet 127.0.0.1 8080`. `nc -v ::1 8080` also failed with connection refused.